### ISSUE TITLE
Logging: log.warn() are not printed to  STDOUT or STDERR

### ISSUE THREAD

**Issue by alon-e** [iotaledger/iri#721]
### Bug description
`log.warn()` does not print to any output.

### Steps To Reproduce
1. open any test with a logger 

2. add 
```
log.info("printed!");
log.warn("not printed!");
log.error("printed!");
```

3. see that `not printed!` is never printed...

**Comment by footloosejava** [comment]
In `src/main/resources/logback.xml` the configuration has a spelling error and some weird stuff (see down last). Also don't know what the 'com' stuff is about. Can change to this and it works properly:

## REPLACE 

```
<?xml version="1.0" encoding="UTF-8"?>
<configuration scan="true" scanPeriod="30 minutes" packagingData="true">
    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
        <target>System.out</target>
        <encoder>
            <pattern>%d{MM/dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
            </pattern>
        </encoder>
    </appender>

    <appender name="STDERR" class="ch.qos.logback.core.ConsoleAppender">
        <target>System.err</target>
        <encoder>
            <pattern>%d{MM/dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
            </pattern>
        </encoder>
    </appender>

    <root level="INFO">
        <appender-ref ref="STDOUT"/>
        <appender-ref ref="STDERR"/>
    </root>
</configuration>
```

## OLD

```
<?xml version="1.0" encoding="UTF-8"?>
<configuration scan="true" scanPeriod="30 minutes" packagingData="true">

	<!-- Appenders -->
	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
	   <filter class="ch.qos.logback.classic.filter.LevelFilter">
            <level>INFO</level>
            <onMatch>ACCEPT</onMatch>
            <onMismatch>DENY</onMismatch>
        </filter>
        
	   <target>System.out</target>
		<encoder>
			<pattern>%d{MM/dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
			</pattern>
		</encoder>
	</appender>

    <!-- 	
	<appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
	   <filter class="ch.qos.logback.classic.filter.LevelFilter">
            <level>DEBUG</level>
            <onMatch>ACCEPT</onMatch>
            <onMismatch>DENY</onMismatch>
        </filter>
        
	   <target>System.out</target>
		<encoder>
			<pattern>%d{MM/dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
			</pattern>
		</encoder>
	</appender>
	-->

	<appender name="STDERR" class="ch.qos.logback.core.ConsoleAppender">
	   <filter class="ch.qos.logback.classic.filter.LevelFilter">
            <level>ERROR</level>
            <onMatch>ACCEPT</onMatch>
            <onMismatch>DENY</onMismatch>
        </filter>
		<target>System.err</target>
		<encoder>
			<pattern>%d{MM/dd HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n
			</pattern>
		</encoder>
	</appender>

	<logger name="com" level="INFO">
		<appender-ref ref="STDOUT" />
	</logger>

	<logger name="com" level="DEBUG">
		<appender-ref ref="STDOUT" />
	</logger>

	<logger name="com" level="ERROR">
		<appender-ref ref="STDERR" />
	</logger>

	<!--  root level="INFO"> <appender-ref ref="STDOUT" /> </root> -->
</configuration>

```

**Comment by footloosejava** [comment]
see revised.

**Comment by GalRogozinski** [comment]
Great @footloosejava !
If you want you want to make a PR, it will be great.
If not we will copy/paste it.

**Comment by jakubcech** [comment]
Fixed by #727
